package.json

```diff
@@ -1813,6 +1813,155 @@
 											}
 										]
 									},
+									{
+										"key": "languageId:cpp",
+										"name": "C++",
+										"type": "bindings",
+										"bindings": [
+											{
+												"key": "=",
+												"name": "+Format",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "=",
+														"name": "Format region or buffer",
+														"type": "command",
+														"command": "editor.action.format"
+													},
+													{
+														"key": "b",
+														"name": "Format buffer",
+														"type": "command",
+														"command": "editor.action.formatDocument"
+													},
+													{
+														"key": "c",
+														"name": "Format changes",
+														"type": "command",
+														"command": "editor.action.formatChanges"
+													},
+													{
+														"key": "s",
+														"name": "Format selection",
+														"type": "command",
+														"command": "editor.action.formatSelection"
+													},
+													{
+														"key": "B",
+														"name": "Format buffer with formatter",
+														"type": "command",
+														"command": "editor.action.formatDocument.multiple"
+													},
+													{
+														"key": "S",
+														"name": "Format selection with formatter",
+														"type": "command",
+														"command": "editor.action.formatSelection.multiple"
+													}
+												]
+											},
+											{
+												"key": "b",
+												"name": "+Backend",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "d",
+														"name": "Reset Database",
+														"type": "command",
+														"command": "C_Cpp.ResetDatabase"
+													},
+													{
+														"key": "w",
+														"name": "Rescan Workspace",
+														"type": "command",
+														"command": "C_Cpp.RescanWorkspace"
+													}
+												]
+											},
+											{
+												"key": "g",
+												"name": "+Go to",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "a",
+														"name": "Switch Header/Source",
+														"type": "command",
+														"command": "C_Cpp.SwitchHeaderSource"
+													},
+													{
+														"key": "d",
+														"name": "Peek declaration",
```

Comment:
Is there any reason it seems like these bindings are inverse of other bindings (e.g. go's bindings)
d is usually go to definition instead of peek
g is usually go to definition instead of peek

Reply:
Got you. I thought vscode usually do a peek if there's more than one references/definitions/declarations when you do a reveal. If that's not the case, I think what you are proposing makes sense.

@MarcoIeni Any thoughts on this?

Reply:
In my limited session with C++, it does "intelligently" show the peek overlay.

@mayanez How about we flip the peek and reveal you have here, so it's more consistent across languages. If we have more complaints, we can always change it 😄

Reply:
Yeah, I find peek annoying, what about moving current bindings to upper case `D` and `G`  and having `d` and `g` to `editor.action.revealDefinition` as golang?

Reply:
Sure I will flip them then. Not a big deal.